sympy/stats/crv.py

```diff
@@ -150,42 +150,38 @@ def __call__(self, *args):
 
 
 class SampleExternalContinuous:
-    """Class consisting of the methods that are used to sample values of random
-    variables from external libraries."""
+    """Base Class to sample values of Continuous random variables from
+    external libraries."""
 
+    def __new__(cls, dist, size, library):
+        _sample_class = _get_sample_class[library]
```

Comment:
```suggestion
        _sample_class = 'SampleContinuous' + upper(library[0]) + library[1:]
```

Reply:
This will require `sympy` to be imported. So, a dictionary could work faster

Reply:
Can you shift `_get_sample_class` dict under this class. It seems faster than `if-else` ladder.

Reply:
Then I need to shift the base class below three library classes as the dictionary will produce `NotDefinedError`, so dictionary should be after defining all classes as present